config/deployment.yml

```diff
@@ -5,6 +5,8 @@ kind: Deployment
 metadata:
   name: kapp-controller
   namespace: #@ data.values.namespace
+  annotations:
+    kapp_controller_version: #@ data.values.kapp_controller_version
```

Comment:
i think we should namespace this annotation e.g. `kapp-controller.carvel.dev/version`